core/src/main/java/org/apache/carbondata/core/compression/BigIntCompressor.java

```diff
@@ -44,6 +44,11 @@ protected Object compressMaxMin(DataType changedDataType, CarbonWriteDataHolder
       Object max) {
     long maxValue = (long) max;
     long[] value = dataHolder.getWritableLongValues();
+    return compressMaxMin(changedDataType, maxValue, value);
+  }
+
+  protected Object compressMaxMin(DataType changedDataType, long maxValue,
```

Comment:
please add description to these compress functions, make it easier to understand by reader